addons/binding/org.openhab.binding.plugwise/src/main/java/org/openhab/binding/plugwise/handler/PlugwiseSenseHandler.java

```diff
@@ -0,0 +1,206 @@
+                message = new SenseBoundariesSetRequestMessage(macAddress, configuration.getHumidityBoundaryMin(),
+                        configuration.getHumidityBoundaryMax(), configuration.getBoundaryAction());
+            } else if (configuration.getBoundaryType() == BoundaryType.TEMPERATURE) {
+                message = new SenseBoundariesSetRequestMessage(macAddress, configuration.getTemperatureBoundaryMin(),
+                        configuration.getTemperatureBoundaryMax(), configuration.getBoundaryAction());
+            } else {
+                message = new SenseBoundariesSetRequestMessage(macAddress);
+            }
+
+            logger.debug("Sending command to update {} ({}) boundary parameters", deviceType, macAddress);
+            sendCommandMessage(message);
+        }
+        if (updateMeasurementInterval) {
+            logger.debug("Sending command to update {} ({}) measurement interval", deviceType, macAddress);
+            sendCommandMessage(new SenseReportIntervalSetRequest(macAddress, configuration.getMeasurementInterval()));
+        }
+        if (updateSleepParameters) {
+            logger.debug("Sending command to update {} ({}) sleep parameters", deviceType, macAddress);
+            sendCommandMessage(new SleepSetRequestMessage(macAddress, configuration.getWakeupDuration(),
+                    configuration.getWakeupInterval()));
+        }
+
+        super.sendConfigurationUpdateCommands();
+    }
+
+    private void setUpdateCommandFlags(PlugwiseSenseConfig oldConfiguration, PlugwiseSenseConfig newConfiguration) {
+        boolean fullUpdate = newConfiguration.isUpdateConfiguration() && !isConfigurationPending();
+        if (fullUpdate) {
+            logger.debug("Updating all configuration properties of {} ({})", deviceType, macAddress);
+        }
+
+        updateBoundaryParameters = fullUpdate
+                || (oldConfiguration != null && !oldConfiguration.equalBoundaryParameters(newConfiguration));
+        if (updateBoundaryParameters) {
+            logger.debug("Updating {} ({}) boundary parameters when online", deviceType, macAddress);
+        }
+
+        updateMeasurementInterval = fullUpdate || (oldConfiguration != null
+                && oldConfiguration.getMeasurementInterval() != newConfiguration.getMeasurementInterval());
+        if (updateMeasurementInterval) {
+            logger.debug("Updating {} ({}) measurement interval when online", deviceType, macAddress);
+        }
+
+        updateSleepParameters = fullUpdate
+                || (oldConfiguration != null && !oldConfiguration.equalSleepParameters(newConfiguration));
+        if (updateSleepParameters) {
+            logger.debug("Updating {} ({}) sleep parameters when online", deviceType, macAddress);
+        }
+    }
+
+    @Override
+    protected void updateConfiguration(Configuration configuration) {
+        PlugwiseSenseConfig oldConfiguration = this.configuration;
```

Comment:
Each Thing type has its own configuration parameters and it needs to know when these are updated so it can send them to the device. Is there any other way besides overriding this method?